standardization/src/main/scala/za/co/absa/enceladus/standardization/StandardizationJob.scala

```diff
@@ -106,6 +106,9 @@ object StandardizationJob {
     Atum.setAdditionalInfo(s"enceladus_info_date" -> cmd.reportDate)
     Atum.setAdditionalInfo(s"enceladus_info_version" -> reportVersion.toString)
 
+    // Add the raw format of the input file(s) to Atum's metadta as well
+    Atum.setAdditionalInfo(s"raw_format" -> cmd.rawFormat)
```

Comment:
Why is there `s` in front of the string?

Reply:
I knew something is wrong with this PR.